pandas/core/frame.py

```diff
@@ -8454,6 +8454,9 @@ def applymap(
             Python function, returns a single value from a single value.
         na_action : {None, 'ignore'}, default None
             If ‘ignore’, propagate NaN values, without passing them to func.
+        **kwargs
+            Additional keyword arguments to pass as keywords arguments to
+            `func`.
```

Comment:
you need to move the versionadded 1.2 which applies to na_action up

and add a versionadded 1.3 for this one